doctor_visits/setup.py

```diff
@@ -11,6 +11,7 @@
     "pytest-cov",
     "pytest",
     "scikit-learn",
+    "dask>=2024.6, <2024.7",
```

Comment:
i think this is the syntax you want...  `dask` uses a 3-tier version format:
```suggestion
    "dask>=2024.6.*,<2024.7.0",
```
also see https://peps.python.org/pep-0440/#version-specifiers